Source/HTTP/WinHttp/winhttp_http_task.cpp

```diff
@@ -823,6 +823,10 @@ void winhttp_http_task::perform_async()
 
 NAMESPACE_XBOX_HTTP_CLIENT_END
 
+void Internal_HCHttpCallInitialize(void* context)
```

Comment:
>context [](start = 41, length = 7)

assert is null

Reply:
does the ios version need updating too?

---
In reply to: [189752159](https://github.com/Microsoft/libHttpClient/pull/165#discussion_r189752159) [](ancestors = 189752159)